docs/swagger/traPortfolio.v1.yaml

```
@@ -0,0 +1,558 @@
+openapi: 3.0.0
+info:
+  title: traPortfolio
+  version: '1.0'
+servers:
+  - url: 'http://localhost:3000'
+paths:
+  /users:
+    get:
+      summary: Your GET endpoint
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/User'
+      operationId: get-users
```

```suggestion
      operationId: getUsers
```
operationIdはキャメルケースにしたい:pray: